examples/tests/multiple_tests.py

```diff
@@ -11,7 +11,6 @@ class MultipleTests(Test):
     so that 'test_foo' and 'testFoo' are test methods,
     but 'division_by_zero' and 'action' are not.
     """
-
```

Comment:
This commit looks good, besides  this little line removal here, which, in my pylint/inspekt versions give style errors.